wireless_modules/mqtt_test/boot.py

```diff
@@ -0,0 +1,27 @@
+from config import ESSID
+from config import PASSWORD
+
+def connect(eesid, password):
+    import network
```

Comment:
Is this some micropython thing where you import inside a function?

Reply:
Yeah it speeds up the code so that you only fetch things when you need them

Reply:
All of the examples use this. I'm not sure how much of a difference it makes though.